go/Dockerfile

```diff
@@ -0,0 +1,43 @@
+
+# syntax=docker.io/docker/dockerfile:1.4
+FROM cartesi/rootfs:0.17.0 as build-stage
+
+ARG GOVERSION=1.20.5.linux-amd64
+
+WORKDIR /opt/build
+
+RUN wget https://go.dev/dl/go${GOVERSION}.tar.gz && tar -C /usr/local -xzf go${GOVERSION}.tar.gz
+
+ENV GOOS=linux
+ENV GOARCH=riscv64
+ENV CGO_ENABLED=1
+ENV CC=riscv64-cartesi-linux-gnu-gcc
```

Comment:
We need to stop using this cross-compiler (and the roots image), and use other standard cross-compilers.